### ISSUE TITLE
Forensics IAM Permission Denied trying to modify Instance Role

### ISSUE THREAD

**Issue by alanty** [aws-samples/eks-security-compromised-cluster-remediation#8]
Under [collecting evidence](https://github.com/aws-samples/eks-security-compromised-cluster-remediation/tree/main/Containment/forensics#collecting-evidence) there is a section to modify the IAM permissions for the Node Role and add the policy below:

```
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "VisualEditor0",
            "Effect": "Allow",
            "Action": [
                "s3:PutObject",
                "s3:GetObject"
            ],
            "Resource": "arn:aws:s3:::clusterstack-forensicsbucket53bd3a1e-ub81jhk20uvk/*"
        }
    ]
}
```

When creating an inline policy or stand alone policy I am receiving the error below:

> User: arn:aws:sts::775288309636:assumed-role/TeamRole/MasterKey is not authorized to perform: iam:CreatePolicy on resource: policy eks-S3upload

and

> User: arn:aws:sts::775288309636:assumed-role/TeamRole/MasterKey is not authorized to perform: iam:PutRolePolicy on resource: role ClusterStack-ng1NodeGroupRole341A083C-JKSXZFE1UL88

**Comment by alanty** [comment]
looks like this may impact only the Console Role from the engine accounts, using the Pre-create Cloud9 Workspace I'm able to attach the policy just fine:

1. Create a file with the IAM policy above, i.e. `S3-up.json`

2. Add this as an inline policy on the Node instance role:
```
aws iam put-role-policy --role-name ClusterStack-ng1NodeGroupRole341A083C-JKSXZFE1UL88 --policy-name EKSS3 --policy-document file://S3-up.json
```

**Comment by jicowan** [comment]
Update EC2 instance profile for node group/worker nodes to include write access to the forensics bucket.

**Comment by mifisignal** [comment]
Fixed in `5895c41`
